server/services/role.js

```diff
@@ -0,0 +1,60 @@
+//  Defines various services for the role object
+
+const Sequelize = require('sequelize');
+// Require sequelize from the connection settings
+const sequelize = require('../../settings/connect');
+
+// Call the role model and specify the arguments.
+const Role = require('../../app/models/role')(sequelize, Sequelize);
+
+sequelize.sync({ });
+
+/**
+ * Create a new role
+ * @param {object} req
+ * @param {function} res // Callback
+ * @returns {boolean} true if created,false otherwise
+ */
+module.exports.createRole = (req, res) => {
+  Role.findOrCreate({
+    where: {
+      title: req.body.title
+    },
+    defaults: {
+      title: req.body.title
+    }
+  })
+  .spread((role, created) => (created)
+    ? res.status(201).send({ message: 'Role Added!' })
+    : res.status(400).send({ message: 'Role Already Exists!' }));
+};
+
+/**
+ * Get role
+ * @param {object} req
+ * @param {function} res // Callback
```

Comment:
Noticed you're referring to `res` object as a `callback`. Shouldn't it be an `object`?